spacify-portal/src/main/java/org/uci/spacifyPortal/utilities/CreateRequest.java

```diff
@@ -0,0 +1,26 @@
+package org.uci.spacifyPortal.utilities;
+
```

Comment:
I have created a CreateRoom object that is being used to take in the user request for the room . Lets use that?